blocks/feeds-source-content-api-block/sources/feeds-content-api.js

```diff
@@ -137,7 +137,7 @@ const resolve = function resolve(key) {
   // if Section append section query to basic query
   const { Section } = key
   if (Section && Section !== '/') {
-    let section = Section.replace(/\/$/, '')
+    let section = Section.replace(/\//g, '')
```

Comment:
You can't do that to a section.  What if they are searching for /sports/high-school/football/